pkg/clockcache/cache_bench_test.go

```diff
@@ -17,7 +17,7 @@ func BenchmarkIntCache(b *testing.B) {
 	b.ReportAllocs()
 	b.ResetTimer()
 	for i := 0; i < b.N; i++ {
-		cache.Insert(keys[i], keys[i])
+		cache.Insert(keys[i], keys[i], 32)
```

Comment:
This is the weird part. How do we know if that's `32`? unsafe tells me `interface` is 16 bits.

Reply:
its for both they key and value

Reply:
this is a test anyway, so shouldn't matter